### PR TITLE
docs: delete .github/ISSUE_TEMPLATE folder

### LINKED ISSUE #1013: Remove .github/ISSUE_TEMPLATE folder
## Description

As a org admin / maintainer,
I need to maintain 1 version of issue templates used in the organization,
so that I can avoid duplicating work and having different versions of the templates, when we want to use the same ones across all projects.

Because the issue templates are now on https://github.com/anitab-org/.github, we don't need to have issue templates present in the repository. Read more about these type of files here https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file

### Update [Required]
- [ ] Delete `.github/ISSUE_TEMPLATE`

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by s4-2**
### Description

Fixes #1013

### Type of Change:
- Documentation

**Code/Quality Assurance Only**

- This change requires a documentation update (software upgrade on readme file)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/quality_assurance.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 name: Quality Assurance task
 about: Provide a group of test cases for the some parts of the project.
+title: ''
+labels: enhancement, good first issue, question
```
Comment (isabelcosta) [inline .github/ISSUE_TEMPLATE/quality_assurance.md]: this is actually very interesting, I did not know we could do this 🤓

**Review by isabelcosta — CHANGES_REQUESTED** [review]
I actually learned something new here 🤓 However the issue you are solving is about deleting the `.github/ISSUE_TEMPLATE` folder completely, because we can use instead the ones from our `.github` repository.

Can you please delete the folder completely and commit again?
And thank you for working on this!

**Comment by s4-2** [comment]
Okay, I will do it now.

**Comment by isabelcosta** [comment]
@s4-2 you can edit these PR changes and commit and push again, no need to create another PR for it. Can you do this on the same branch please?

**Comment by isabelcosta** [comment]
@s4-2 let me know if you have any doubt about it, I can still try to help

**Review by isabelcosta — APPROVED** [review]
Awesome! Great work @s4-2 and thank you for following my suggestions :)
Thank you so much for working on this 🤗

**Comment by s4-2** [comment]
@isabelcosta  I have not yet use push command, can you guide me how I can do this please.

**Comment by isabelcosta** [comment]
It's already pushed @s4-2 :) you probably did this via GitHub interface and in that case, it is already in the remote repo (same as pushed). your work is done :) I just need another approval and then I can merge.

**Comment by s4-2** [comment]
Thank you @isabelcosta, I am happy that you liked my work.
I would like to Thank  @epicadk  and @isabelcosta  for your continuous guidance and help.

**Review by vj-codes — APPROVED** [review]
LGTM 🎉 
Thank you for your contribution @s4-2

**Comment by vj-codes** [comment]
@s4-2 just update fixes part of the description with issue no ie #1013 instead of PR no to keep a better track of records.

**Comment by vj-codes** [comment]
Adding label ready to merge since no part of code has been changed.

**Review by epicadk — APPROVED** [review]
LGTM. Thankyou for your contribution. : )

**Pull request merged**